config/config.toml

```
@@ -38,6 +38,9 @@ secrets = "/etc/voucher/secrets.production.ejson"
 address = "localhost:6060"
 
 [statsd]
+# with a local aggregator:
+backend = statsd
```

```suggestion
backend = "statsd"
```

the quotes are not needed in the end?